app/views/home/index.html.erb

```diff
@@ -0,0 +1,54 @@
+<section class="home-page">
+<h1>My Budget</h1>
```

Comment:
- [Optional] After opening up your app locally initially, I thought your app is not working. I needed to scroll right down below before seeing the title. This creates a bad user experience while using your app. I suggest you fix the issue. Check the screenshot below:
![Screenshot from 2023-05-20 10-59-04](https://github.com/Salwa99/My-Budget/assets/63087491/2edad886-d95f-4d51-9c6e-b70e62dba7ef)